samplestore/src/main/java/com/stripe/samplestore/PaymentActivity.java

```diff
@@ -13,36 +13,27 @@
 import android.support.v4.content.LocalBroadcastManager;
 import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
-import android.text.TextUtils;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.widget.Button;
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
-import com.google.android.gms.identity.intents.model.UserAddress;
-import com.google.android.gms.wallet.Cart;
-import com.google.android.gms.wallet.LineItem;
 import com.jakewharton.rxbinding.view.RxView;
 import com.stripe.android.CustomerSession;
-import com.stripe.android.PaymentConfiguration;
+import com.stripe.android.PayWithGoogleUtils;
```

Comment:
Yes, we're using the PayWithGoogleUtils class in a project that (at the moment) cannot pay with google. But it's setting up for the future, and it shows how to use the function.